package.json

```diff
@@ -16,18 +16,26 @@
     "popsicle": "^9.2.0"
   },
   "devDependencies": {
-    "eslint": "^4.17.0"
+    "coveralls": "^3.0.0",
+    "eslint": "^4.17.0",
+    "faucet": "0.0.1",
+    "nyc": "^11.4.1",
+    "tape": "^4.8.0"
   },
   "homepage": "https://github.com/mozilla/blurts-server#readme",
   "license": "ISC",
   "main": "server.js",
+  "nyc": {
+      "all": true
+  },
   "private": true,
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mozilla/blurts-server.git"
   },
   "scripts": {
     "lint": "eslint .",
-    "start": "node server.js"
+    "start": "node server.js",
+    "test": "eslint . && nyc tape tests/**/*.js | faucet && nyc report --reporter=text-lcov | coveralls"
```

Comment:
Instead of calling ESLint here, we should instead call `npm run lint` so we don't need to update linting configs in 2 places.
Also, not sure if we want to scrap linting here and just use a `"pretest": "npm run lint"` hook.

Reply:
When I used `npm run lint` here, it messed up Travis. `npm run lint` on Travis made `eslint` scan `node_modules/*` too, which threw all kinds of errors. I searched around and found a few bug reports, but didn't find a fix.

Reply:
This ~~interests~~ fascinates me, and I wonder if it's related to you adding a **.npmignore** file. Although I can't find any docs stating that if ESLint cannot find an **.eslintignore** file, it will revert back to using **.npmignore** _then_ **.gitignore**.

We _could_ add our own **.eslintignore** file with node_modules in it, but that should be the default behavior, which is why this is so bizarre. But I'm fine to land this whenever and fix that later. I guess the quick test would maybe be to try adding "node_modules" to the **.npmignore** file.